pkg/operator/controller/ingress/deployment_test.go

```diff
@@ -68,6 +68,20 @@ func checkDeploymentHasContainer(t *testing.T, deployment *appsv1.Deployment, na
 	}
 }
 
+func checkRouterContainerSecurityContext(t *testing.T, deployment *appsv1.Deployment) {
+	routerContainerName := "router"
```

Comment:
Can you make this a constant and move it to the top of the file?
```go
const (
	routerContainerName = "router"
)
```
We should also update other tests to use the named constant, but that can be left for future PRs.

Reply:
Fixed!